tasks/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: Perform parameter compatibility check
+  fail:
+    msg: "Subcription manager should run only when ansible_distribution == 'Red Hat Enterprise Linux'"
```

Comment:
I'd rephrase, it seems to me too "developer specific" :-)
```
msg: "Subcription manager could be used only on Red Hat Enterprise Linux"
```

Reply:
Done